.github/workflows/build_and_test.yaml

```diff
@@ -17,10 +18,7 @@ on:
 jobs:
   lint:
     name: Lint
-    strategy:
-      matrix:
-        os: [ubuntu-22.04]
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}](runs-on: ${{ inputs.runner || 'ubuntu-22.04' }})
```

Comment:
```suggestion
    runs-on: ${{ inputs.runner || 'ubuntu-22.04' }})
```
My fault

Reply:
I had to revert this to my initial as it was failing the run. The matrix run is not particularly useful with the current setup because the compile and run step all occurs within a docker. Changing the runner has not real impact on the outcome.